src/_pytest/nodes.py

```diff
@@ -445,7 +445,7 @@ def _repr_failure_py(
         if isinstance(excinfo.value, ConftestImportFailure):
             excinfo = ExceptionInfo.from_exc_info(excinfo.value.excinfo)
         if isinstance(excinfo.value, fail.Exception):
-            if not excinfo.value.pytrace:
+            if not excinfo.value.pytrace and style != "line":
```

Comment:
The reason `"value"` doesn't end up working for this seems to be this line

https://github.com/pytest-dev/pytest/blob/7834b3b07f7a7b2409331a8f14141c6ee8546215/src/_pytest/_code/code.py#L980-L982

which says that for `style = "value"`, the reprcrash is None. The reprcrash is what `--tb=line` uses:

https://github.com/pytest-dev/pytest/blob/7834b3b07f7a7b2409331a8f14141c6ee8546215/src/_pytest/terminal.py#L1011-L1014

this code is currently the only place where `--tb=line` condition appears which seems like a nice property to me.

So what I wonder is why "value" doesn't want a reprcrash. This comes from 94c7b8b47cd6b5b14f463731e473929b42881073 which introduced "line". I think the idea was to maintain the previous behavior where there was also no reprcrash for `pytrace=False`.

What is `reprcrash` anyway? TBH I'm not sure (pytest is full of mysteries like that). [...After looking at the code...] It seems to be used for the following:

1. For `--tb=line`. This is the one we want to affect so :heavy_check_mark: 
2. Shown in the short test summary info. Seems good to show the reason message here so :heavy_check_mark: .
3. In junitxml output. Not an expert but the change seems pretty minor - adding the `Failed: ` prefix. :heavy_check_mark: 
4. Some other irrelevant stuff to "value" (skips, KeyboardInterruprt). :heavy_check_mark: 

So I think instead of this change, we can replace

```py
reprcrash: Optional[ReprFileLocation] = ( 
    excinfo_._getreprcrash() if self.style != "value" else None 
) 
```

to

```py
reprcrash = excinfo_._getreprcrash()
```

Can you try this?

Reply:
Didn't realize we were on the cutting edge of development here:
https://github.com/pytest-dev/pytest/blob/7834b3b07f7a7b2409331a8f14141c6ee8546215/src/_pytest/_code/code.py#L980-L982
was added as of the latest release, one day after I authored my PR!

I removed my changes and changed those three lines to
```
reprcrash = excinfo_._getreprcrash()
```
and all of the tests, including the one I added, passed! Pushing the changes in a new commit now.

Reply:
Yes we've recently had some crash due to reprcrash :)